src/utils/cmrIngest.js

```diff
@@ -14,7 +12,12 @@ import { pickIgnoringCase } from './pickIgnoringCase'
  * @param {Object} headers Headers to send to CMR
  * @param {String} ingestPath CMR path to call to ingest concept
  */
-export const cmrIngest = async (conceptType, data, headers, ingestPath) => {
+export const cmrIngest = ({
```

Comment:
Update the JS doc comment above to represent these parameters coming in through an object, look at `getCollectionsById.js`